### PR TITLE
feat(ui) Show Acryl information with button and banner behind flag

### PR DESCRIPTION
**Pull request by chriscollins3456**
Creates a few desired Managed DataHub / Acryl CTAs that are hidden behind a feature flag that is default off. We will turn this on in our demo site to show a button to book a demo and a banner on the home page.

image

image

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/home/AcrylDemoBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import styled from 'styled-components';
+import AcrylLogo from '../../images/acryl-light-mark.svg';
+
+const BannerWrapper = styled.div`
+    padding: 12px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: #262626;
+    background-color: #e6f4ff;
```
Comment (joshuaeilers) [inline datahub-web-react/src/app/home/AcrylDemoBanner.tsx:12]: May be nice to put these into a constants, but I bet they're only used here for now. If so, it's fine the way it is.

File: datahub-web-react/src/app/home/AcrylDemoBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+const TextWrapper = styled.div`
+    font-size: 14px;
+`;
+
+const Title = styled.div`
+    font-weight: 700;
+`;
+
+const StyledLink = styled(Link)`
+    color: #1890ff;
```
Comment (joshuaeilers) [inline datahub-web-react/src/app/home/AcrylDemoBanner.tsx:32]: Same question about this color.

File: datahub-web-react/src/app/useAppConfig.ts [hunk 3]
```diff
@@ -7,3 +7,8 @@ import { AppConfigContext } from '../appConfigContext';
 export function useAppConfig() {
     return useContext(AppConfigContext);
 }
+
+export function useIsShowAcrylInfoEnabled() {
+    const appConfig = useAppConfig();
+    return appConfig.config.featureFlags.showAcrylInfo;
+}
```
Comment (joshuaeilers) [inline datahub-web-react/src/app/useAppConfig.ts:14]: Thank you for making this a shared hook ❤️

File: metadata-service/factories/src/main/resources/application.yml [hunk 4]
```diff
@@ -289,6 +289,7 @@ featureFlags:
   showBrowseV2: ${SHOW_BROWSE_V2:true} # Enables showing the browse v2 sidebar experience.
   preProcessHooks:
     uiEnabled: ${PRE_PROCESS_HOOKS_UI_ENABLED:true} # Circumvents Kafka for processing index updates for UI changes sourced from GraphQL to avoid processing delays
+  showAcrylInfo: ${SHOW_ACRYL_INFO:true} # Show different CTAs within DataHub around moving to Managed DataHub. Set to true for the demo site.
```
Comment (joshuaeilers) [inline metadata-service/factories/src/main/resources/application.yml:292]: Default to false here probably?

**Review by joshuaeilers — APPROVED** [review]
Looks good. Only one potential issue with the feature flag default, otherwise looks great.

File: metadata-service/factories/src/main/resources/application.yml (hunk 4 above)
Comment (chriscollins3456) [reply metadata-service/factories/src/main/resources/application.yml:292]: oh man amazing call haha woops! i forgot to switch it back

**Pull request merged**